recipes-domx/meta-xt-images-vgpu/recipes-core/llvm/llvmpvr_3.7.0.bb

```diff
@@ -0,0 +1,116 @@
+DESCRIPTION = "The Low Level Virtual Machine"
```

Comment:
Looks like not so much of LLVM is left there... Also `llvmpvr_3.7.0.bb` suggests that this is some special build for PVR.
I think that this should be mentioned in description.